platform-sdk/swirlds-logging/src/jmh/java/com/swirlds/logging/benchmark/swirldslog/SwirldsLogLoggingBenchmarkConfig.java

```diff
@@ -47,15 +47,14 @@ public class SwirldsLogLoggingBenchmarkConfig implements LoggingBenchmarkConfig<
                 .withConverter(new MarkerStateConverter())
                 .withValue("logging.level", "trace")
                 .withValue("logging.handler.file.type", "file")
-                .withValue("logging.handler.file.active", "true")
+                .withValue("logging.handler.file.enabled", "true")
```

Comment:
we should fix this property in unit tests too.

Reply:
will do in a follow-up